app/services/cafe_service.rb

```diff
@@ -0,0 +1,49 @@
+# encoding: UTF-8
+module CafeService
+  def self.setup(start, stop, day, lp)
+    CafeShift.create!(preview(start, stop, day, lp))
+  end
+
+  def self.preview(start, stop, day, lp)
+    shifts = []
+    (start..stop).each do |lv|
+      (0..4).each do
+        shifts << { start: day, lp: lp, pass: 1, lv: lv }
+        shifts << { start: day, lp: lp, pass: 2, lv: lv }
+        shifts << { start: day + 2.hours, lp: lp, pass: 3, lv: lv }
+        shifts << { start: day + 2.hours, lp: lp, pass: 4, lv: lv }
+        day += 1.days
+      end
+      day += 2.days
+    end
+    shifts
+  end
+
+  def self.create_worker(shift, param)
+    puts param
+    shift.build_cafe_worker(param)
+
+    if shift.cafe_worker.save!
+      self.cafe_worker_groups(shift, param)
+      true
+    else
+      false
+    end
+  end
+
+  def self.update_worker(shift, param)
```

Comment:
Unused method argument - `shift`. If it's necessary, use `_` or `_shift` as an argument name to indicate that it won't be used. You can also write as `update_worker(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `param`. If it's necessary, use `_` or `_param` as an argument name to indicate that it won't be used. You can also write as `update_worker(*)` if you want the method to accept any arguments but don't care about them.